src/Translator.zig

```diff
@@ -563,6 +565,7 @@ fn transRecordDecl(t: *Translator, scope: *Scope, record_qt: QualType) Error!voi
         // mangled name is of no real use here.
         if (!is_unnamed and !t.global_names.contains(bare_name) and t.weak_global_names.contains(bare_name))
             try t.alias_list.append(t.gpa, .{ .alias = bare_name, .name = name });
+        try t.global_scope.addContainerDecl(record_qt, node);
```

Comment:
try root.container_member_fns_map.put(root.translator.gpa, container_qt, container_member_fns);
```suggestion
        try t.global_scope.container_member_fns_map.put(t.gpa, record_qt, .{
            .container_decl = init_node,
        });
```